### DISCUSSION TITLE
Downloaded items forced import into lazer

### DISCUSSION THREAD

**Discussion by optionaldata0** [ppy/osu#29625] [Q&A]
### Type

Other

### Bug description

Both stable and lazer builds are installed.
When importing an external file, like a skin or beatmap, the selected file will ALWAYS import into lazer, even when specifically selecting to open the file with the stable build .exe. See video. The .exe shown in the video is the stable build.

Expected behaviour is to import the skin into the build which is selected, not always into lazer.

### Screenshots or videos

https://github.com/user-attachments/assets/2121b15d-5d18-4f79-8242-a052ff9222c1

### Version

2024.817.0

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16781552/compressed-logs.zip)

**Comment by peppy** [comment]
Lazer will currently take file associations over when it's run.
